src/modules/ROOT/pages/changelog/202303/02.json

```diff
@@ -0,0 +1,233 @@
+        "https://github.com/cortezaproject/corteza/commit/46f5455bd0ae67b9de2159f9d04b054a711832e1"
+      ]
+    },
+    {
+      "what": [
+        "page layouts to namespace export resources."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/commit/4ad121747d136f76aa8701f07840c9b75c4aef37"
+      ]
+    }
+  ],
+  "changed": [
+    {
+      "what": [
+        "{APP_NAME_ADMIN} role and user picker with vue-select."
+      ],
+      "why": [
+        "to be consistent with the rest of the pickers used in {PRODUCT_NAME}."
+      ],
+      "how": [
+        "by adding vue-select to role and member picker components."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/1194"
+      ]
+    },
+    {
+      "what": [
+        "the implementation for reminders in {APP_NAME_ADMIN}."
+      ],
+      "why": [
+        "to be consistent with the rest of the {PRODUCT_NAME} UI and improve the UE."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/1239"
+      ]
+    },
+    {
+      "what": [
+        "the feedback email domain from `crust.tech` to `cortezaproject.org`."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/commit/0072a4bf4004f667bc98c6543392c265e4c3991e"
+      ]
+    },
+    {
+      "what": [
+        "discarded the snoozed flag when the reminded at flag is changed in reminders."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/commit/256dd59b83f146b302b44eb808649570f326f37c"
+      ]
+    },
+    {
+      "what": [
+        "the format of displaying the total number of record list items."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/commit/0fcbec0b9a6ec652d206a7fbad7639c36f50c10b"
+      ]
+    },
+    {
+      "what": [
+        "block magnify query param."
+      ],
+      "why": [
+        "to ensure that any future code implementations don't collide with this functionality."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/commit/5fde4c99ee6e5a9c9d95e5bf9a3f5623a41743ce"
+      ]
+    }
+  ],
+  "fixed": [
+    {
+      "what": [
+        "multiline privacy request comments displaying as inline comments."
+      ],
+      "how": [
+        "by adding support for multiline comments in privacy requests."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/1214"
+      ]
+    },
+    {
+      "what": [
+        "{APP_NAME_COMPOSE} broken public icon url for page, namespace, and custom app icons after changing the Auth JWT secret."
+      ],
+      "how": [
+        "by adding exception for the namespace icons, and page/icon attachments."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/1219"
+      ]
+    },
+    {
+      "what": [
+        "available and selected fields not scrollable in safari due to set max height in the item picker component."
```

Comment:
On what interface was this? Module edit I assume?

Reply:
everywhere it's used